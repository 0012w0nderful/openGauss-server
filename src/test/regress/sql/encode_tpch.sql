/*
################################################################################
# TESTCASE NAME : encode_tpch
# COMPONENT(S)  : plan encoding
# MODIFIED BY   : WHO            WHEN          COMMENT
#               : -------------- ------------- ---------------------------------
#               :                1-9-2019      created
################################################################################
*/
\c postgres

select create_wlm_operator_info(1);

select count(*) from gs_wlm_plan_operator_info;

select gather_encoding_info('regression');

select count(*) from gs_wlm_plan_encoding_table;
