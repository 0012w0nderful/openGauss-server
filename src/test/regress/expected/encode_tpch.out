/*
################################################################################
# TESTCASE NAME : encode_tpch
# COMPONENT(S)  : plan encoding
# MODIFIED BY   : WHO            WHEN          COMMENT
#               : -------------- ------------- ---------------------------------
#               :                1-9-2019      created
################################################################################
*/
\c postgres
select create_wlm_operator_info(1);
 create_wlm_operator_info 
--------------------------
                        0
(1 row)

select count(*) from gs_wlm_plan_operator_info;
 count 
-------
--?   .*
(1 row)

select gather_encoding_info('regression');
 gather_encoding_info 
----------------------
                    0
(1 row)

select count(*) from gs_wlm_plan_encoding_table;
 count 
-------
--?   .*
(1 row)

